website/templates/project/statistics.mako

```diff
@@ -5,6 +5,24 @@
   <h2 class="text-300">Analytics</h2>
 </div>
 
+<div id="hiddenFlag"></div>
+<script src="/static/js/ads.js"></script>
+<script>
+  if( window.canRunAds === undefined ){
+    var banner = document.createElement('div');
+    banner.className += 'm-b-md p-md osf-box-lt box-round text-center';
```

Comment:
Not quite what I was saying. You have the hiddenFlag div above, so just put the content in there rather than making it with Javascript.

In base.css, you could make a .adBlockVisible class. That could look something like:
```css
.ad_block_visible: {
    display: hidden
}
```

Then your hidden flag div could look like
```html
<div id="hiddenFlag" class='ad_block_visible m-b-md p-md osf-box-lt box-round text-center' >
    The use of adblocking software may prevent site analytics from loading properly.
    <!-- Look at <a href="http://help.osf.io/page/does/not/exist/yet">our help pages</a> for more info. -->
</div>
````

And your javascript would change display: hidden to display: block. Something in that range? This way HTML is HTML, CSS is CSS, and all the javascript does is make it visible.